specification/postgresql/resource-manager/Microsoft.DBforPostgreSQL/stable/2018-06-01/examples/WaitStatisticsGet.json

```diff
@@ -0,0 +1,29 @@
+{
+  "parameters": {
+    "api-version": "2018-06-01",
+    "subscriptionId": "ffffffff-ffff-ffff-ffff-ffffffffffff",
+    "resourceGroupName": "testResourceGroupName",
+    "serverName": "testServerName",
+    "waitStatisticsId": "636927606000000000-636927615000000000-send-wait/io/socket/sql/client_connection-2--0"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/ffffffff-ffff-ffff-ffff-ffffffffffff/resourceGroups/testResourceGroupName/providers/Microsoft.DBforPostgreSQL/servers/testServerName/waitStatistics/636927606000000000-636927615000000000-send-wait/io/socket/sql/client_connection-2--0",
+        "name": "636927606000000000-636927615000000000-send-wait/io/socket/sql/client_connection-2--0",
```

Comment:
resource names cannot have '/' characters in them.  See https://armwiki.azurewebsites.net/introduction/concepts/regions.html?q=arm%20regions